src/components/Collection/Series/EditSeriesTabs/NameTab.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { mdiMagnify, mdiPencilCircleOutline } from '@mdi/js';
+
+import Input from '@/components/Input/Input';
+import { useGetSeriesQuery } from '@/core/rtkQuery/splitV3Api/seriesApi';
+
+type Props = {
+  seriesId: number;
+};
+
+const NameTab = ({ seriesId }: Props) => {
+  const [name, setName] = useState('');
```

Comment:
This doesn't look like it's hooked up yet. Are we missing an endpoint?

Reply:
yes